docs/core/tools/dotnet-new.md

```diff
@@ -58,714 +28,688 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
+| ASP.NET Core Web App (Model-View-Controller) | [mvc](#web-options)             | [C#], F#     | Web/MVC                               | 1.0        |
+| ASP.NET Core Web App                         | [webapp, razor](#web-options)   | [C#]         | Web/MVC/Razor Pages                   | 2.2, 2.0   |
+| ASP.NET Core with Angular                    | [angular](#spa)                 | [C#]         | Web/MVC/SPA                           | 2.0        |
+| ASP.NET Core with React.js                   | [react](#spa)                   | [C#]         | Web/MVC/SPA                           | 2.0        |
+| ASP.NET Core with React.js and Redux         | [reactredux](#reactredux)       | [C#]         | Web/MVC/SPA                           | 2.0        |
+| Razor Class Library                          | [razorclasslib](#razorclasslib) | [C#]         | Web/Razor/Library/Razor Class Library | 2.1        |
+| ASP.NET Core Web API                         | [webapi](#webapi)               | [C#], F#     | Web/WebAPI                            | 1.0        |
+| ASP.NET Core gRPC Service                    | [grpc](#web-others)             | [C#]         | Web/gRPC                              | 3.0        |
+| Protocol Buffer File                         | [proto](#namespace)             |              | Web/gRPC                              | 3.0        |
+| dotnet gitignore file                        | `gitignore`                     |              | Config                                | 3.0        |
+| global.json file                             | [globaljson](#globaljson)       |              | Config                                | 2.0        |
+| NuGet Config                                 | `nugetconfig`                   |              | Config                                | 1.0        |
+| dotnet local tool manifest file              | `tool-manifest`                 |              | Config                                | 3.0        |
+| Web Config                                   | `webconfig`                     |              | Config                                | 1.0        |
+| Solution File                                | `sln`                           |              | Solution                              | 1.0        |
 
 ## Options
 
-# [.NET Core 2.2](#tab/netcore22)
-
-`--dry-run`
-
-Displays a summary of what would happen if the given command were run if it would result in a template creation.
-
-`--force`
-
-Forces content to be generated even if it would change existing files. This is required when the output directory already contains a project.
-
-`-h|--help`
+- **`--dry-run`**
 
-Prints out help for the command. It can be invoked for the `dotnet new` command itself or for any template, such as `dotnet new mvc --help`.
+  Displays a summary of what would happen if the given command were run if it would result in a template creation. Available since .NET Core 2.2 SDK.
```

Comment:
Good question! Running this, it just shows the result of what would happen if you were to run the command. @KathleenDollard @dsplaisted this is the same text shown by --help. It would be good to fix this because indeed it's confusing.